pkg/descheduler/evictions/evictions.go

```diff
@@ -108,6 +423,46 @@ func (pe *PodEvictor) SetClient(client clientset.Interface) {
 	pe.client = client
 }
 
+func (pe *PodEvictor) evictionRequestsTotal() uint {
```

Comment:
I forgot what was the consensus last time, but do we want to introduce new metrics for this feature? https://github.com/kubernetes-sigs/descheduler/blob/master/keps/1397-evictions-in-background/README.md#metrics

Reply:
We do. Yet, not as part of this PR. I will have a follow up for all other changes.